concordium-sdk/pom.xml

```diff
@@ -71,6 +71,11 @@
             <artifactId>jackson-datatype-jsr310</artifactId>
             <version>2.13.2</version>
         </dependency>
+        <dependency>
```

Comment:
What is the reason for not just using `InetAddress`?

If no particular reason exists, then I would prefer if we just use `InetAddress` and avoid having a dependency for this logic.

Reply:
done